Sources/OktaIdx/Capabilities/IDXCapability.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright (c) 2021-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import Foundation
+
+public protocol AuthenticatorCapability {
+}
+
```

Comment:
Worth adding some documentation for these new types?

Reply:
Yes, I'll be adding more docs this afternoon. Dotting I's, crossing T's, etc. 

Largely these protocols are to define which capabilities are applicable to authenticators vs remediations, but in my mind, everything that's public must be documented.